cfme/base/ui.py

```diff
@@ -62,3 +107,612 @@ class Dashboard(CFMENavigateStep):
+        TAB_NAME = "Workers"
+
+    @View.nested
+    class customlogos(Tab):  # noqa
+        TAB_NAME = "Custom Logos"
+
+    @View.nested
+    class advanced(Tab):  # noqa
+        TAB_NAME = "Advanced"
+
+
+@navigator.register(Server)
+class Details(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Configuration')
+
+    def step(self):
+        self.view.accordions.settings.tree.click_path(
+            self.obj.zone.region.settings_string,
+            "Zones",
+            "Zone: {} (current)".format(self.obj.zone.description),
+            "Server: {} [{}] (current)".format(self.obj.name,
+                self.obj.sid))
+
+
+@navigator.register(Server, 'Server')
+class ServerDetails(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        self.view.server.select()
+
+
+@navigator.register(Server)
+class Authentication(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        self.view.authentication.select()
+
+
+@navigator.register(Server)
+class Workers(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        self.view.workers.select()
+
+
+@navigator.register(Server)
+class CustomLogos(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        self.view.customlogos.select()
+
+
+@navigator.register(Server)
+class Advanced(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        self.view.advanced.select()
+
+
+class ServerDiagnosticsView(ConfigurationView):
+    @View.nested
+    class summary(Tab):  # noqa
+        TAB_NAME = "Summary"
+
+    @View.nested
+    class workers(Tab):  # noqa
+        TAB_NAME = "Workers"
+
+    @View.nested
+    class collectlogs(Tab):  # noqa
+        TAB_NAME = "Collect Logs"
+
+    @View.nested
+    class cfmelog(Tab):  # noqa
+        TAB_NAME = "CFME Log"
+
+    @View.nested
+    class auditlog(Tab):  # noqa
+        TAB_NAME = "Audit Log"
+
+    @View.nested
+    class productionlog(Tab):  # noqa
+        TAB_NAME = "Production Log"
+
+    @View.nested
+    class utilization(Tab):  # noqa
+        TAB_NAME = "Utilization"
+
+    @View.nested
+    class timelines(Tab):  # noqa
+        TAB_NAME = "Timelines"
+
+    configuration = Dropdown('Configuration')
+
+
+@navigator.register(Server)
+class Diagnostics(CFMENavigateStep):
+    VIEW = ServerDiagnosticsView
+    prerequisite = NavigateToSibling('Configuration')
+
+    def step(self):
+        self.view.accordions.diagnostics.tree.click_path(
+            self.obj.zone.region.settings_string,
+            "Zone: {} (current)".format(self.obj.zone.description),
+            "Server: {} [{}] (current)".format(
+                self.obj.name, self.obj.sid))
+
+
+@navigator.register(Server)
```

Comment:
Yes, I've been toying with the idea of making navmazing take either a string or a list, so that you can have aliases, but I didn't come to a firm conclusion yet.